fe/fe-core/src/test/java/com/starrocks/analysis/ShowCatalogsStmtTest.java

```diff
@@ -59,10 +59,10 @@ public void testShowCatalogsNormal() throws AnalysisException, DdlException {
         ShowExecutor executor = new ShowExecutor(ctx, stmt);
         ShowResultSet resultSet = executor.execute();
         ShowResultSetMetaData metaData = resultSet.getMetaData();
-        Assert.assertEquals(metaData.getColumn(0).getName(), "Catalog");
-        Assert.assertEquals(metaData.getColumn(1).getName(), "Type");
-        Assert.assertEquals(metaData.getColumn(2).getName(), "Comment");
-        Assert.assertEquals(resultSet.getResultRows().get(0).toString(), "[default, default, internal catalog]");
-        Assert.assertEquals(resultSet.getResultRows().get(1).toString(), "[hive_catalog_1, hive, hive_catalog]");
+        Assert.assertEquals("Catalog", metaData.getColumn(0).getName());
```

Comment:
@adzfolc Please remember how this method is used :)

Reply:
> @adzfolc Please remember how this method is used :)

got